CMake/itkFFTImageFilterInitFactoryRegisterManager.h.in

```diff
@@ -16,8 +16,8 @@
  *
  *=========================================================================*/
 
-#ifndef itkFFTImageFilterFactoryRegisterManager_h
-#define itkFFTImageFilterFactoryRegisterManager_h
+#ifndef itkFFTImageFilterInitFactoryRegisterManager_h
+#define itkFFTImageFilterInitFactoryRegisterManager_h
```

Comment:
`Init` / `INIT` needs to also be added to the appropriate locations in CMake and Python.

Reply:
Good catch, will fix